.github/workflows/examples-cc13x2x7_26x2x7.yaml

```diff
@@ -29,13 +29,12 @@ jobs:
 
         env:
             BUILD_TYPE: gn_cc26x2x7
-            TI_SYSCONFIG_ROOT: /opt/ti/sysconfig_1.11.0
 
         runs-on: ubuntu-latest
         if: github.actor != 'restyled-io[bot]'
 
         container:
-            image: connectedhomeip/chip-build:latest
+            image: connectedhomeip/chip-build-ti:latest
```

Comment:
Please fix the version rather than latest: as we update SDKs we do not want to break old builds

Reply:
done, setting to 0.5.61